Haskell/Elements/Simplicity/Elements/DataTypes.hs

```diff
@@ -689,3 +690,49 @@ outputFee txo = do
   Explicit assetId <- Just . view (under asset) $ txoAsset txo
   Explicit value <- Just . view (under amount) $ txoAmount txo
   return (assetId, value)
+
+-- | The (explicit) asset ID for liquid BTC.
+-- Note that this asset is specific to the production liquid blockchain.
+lBtcAsset :: Hash256
+lBtcAsset | result == review (over le256) 0x6f0279e9ed041c3d710a9f57d0c02928416460c4b722ae3457a11eec381c526d = result
```

Comment:
I keep getting confused by the reversed asset ids. To reiterate, the C code and the Simplicity specification use the order where the LBTC id starts with `0x6d`. The Haskell code uses the order where the same id starts with `0x6f`. I think elementsd also returns `0x6f` first.

Reply:
Elementsd only *displays* asset ids with 0x6f first.  Internally both Simplicity and Elementsd being with 0x6d.

Reply:
the `le` in `review (over le256)` stands for "little endian", meaning it parses the integer in little endian order to create the hash value.

Reply:
But yes, part of the purpose of this code is to compute the hash value by actually hashing the input, thus guaranteeing that the byte order is internally correct (as the byte-reversed version would have no known preimage), and then validating that the hash value matches the expected value.  (And if the expected value were incorrectly written backwards, there would be no way to get this procedure to work).

Reply:
That makes sense. Big endian is what humans are used to seeing and what Simplicity uses. The Haskell code uses little endian because the literal bytes are processed by something else and the output would be different for big endian. elementsd usually prints values in little endian. Thanks for clarifying.